photon-core/src/main/java/org/photonvision/vision/target/TrackedTarget.java

```diff
@@ -364,6 +366,9 @@ public boolean hasSubContours() {
     }
 
     public Transform3d getBestCameraToTarget3d() {
+        if (m_bestCameraToTarget3d == new Transform3d()) {
+            DriverStation.reportWarning("Camera has not been calibrated.", false);
```

Comment:
This warning might be spammed. Also, this transform won't be set if the pipeline isn't in 3d mode -- you don't know which reason it is. 

This functionality in c++ too would be good

Reply:
> Also, this transform won't be set if the pipeline isn't in 3d mode
Should I add a note about that in the warning?